e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1

```diff
@@ -433,41 +479,38 @@ az keyvault secret set --vault-name $secretsKeyVaultName --name "LA-AAD-APP-CERT
 az keyvault secret set --vault-name $secretsKeyVaultName --name "DPS-GLOBALDEVICEENDPOINT-INVALIDCERT" --value "invalidcertgde1.westus.cloudapp.azure.com" --output none
 
 ###################################################################################################################################
-#Run docker containers for TPM simulators and Proxy
+# Run docker containers for TPM simulators and Proxy
 ###################################################################################################################################
 
-if (-not (docker images -q aziotbld/testtpm)) {
+if (-not (docker images -q aziotbld/testtpm))
+{
     Write-Host "Setting up docker container for TPM simulator"
     docker run -d --restart unless-stopped --name azure-iot-tpmsim -p 127.0.0.1:2321:2321 -p 127.0.0.1:2322:2322 aziotbld/testtpm
 }
 
-if (-not (docker images -q aziotbld/testproxy)) {
+if (-not (docker images -q aziotbld/testproxy))
+{
     Write-Host "Setting up docker container for Proxy"
     docker run -d --restart unless-stopped --name azure-iot-tinyproxy -p 127.0.0.1:8888:8888 aziotbld/testproxy
 }
 
 ############################################################################################################################
-# Cleanup certs and files created by the script
+# Clean up certs and files created by the script
 ############################################################################################################################
-Write-Host("`nCleanup certificates and files created by the script")
-Get-ChildItem "Cert:\LocalMachine\My" | Where-Object { $_.Issuer.Contains("CN=$subjectPrefix") } | Remove-Item
-Get-ChildItem "Cert:\LocalMachine\My" | Where-Object { $_.Issuer.Contains("CN=xdevice1") } | Remove-Item
-Get-ChildItem "Cert:\LocalMachine\My" | Where-Object { $_.Issuer.Contains("CN=iothubx509device1") } | Remove-Item
-Get-ChildItem .\ | Where-Object { $_.Name.EndsWith(".pfx") } | Remove-Item
-Get-ChildItem .\ | Where-Object { $_.Name.EndsWith(".cer") } | Remove-Item
-Get-ChildItem .\ | Where-Object { $_.Name.EndsWith(".p7b") } | Remove-Item
+CleanUp-Certs
 
-#Creating a file to run to load environment variables
-$currentDir = Get-Location
-Write-Host "`nOutput in '$currentDir\iotPowershellConfig.ps1'"
-$file = New-Item  -Path $currentDir -Name "iotPowershellConfig.ps1" -ItemType "file" -Force
-Add-Content -Path $file.PSPath -Value "Invoke-Expression `".\LoadEnvironmentVariablesFromKeyVault.ps1 -SubscriptionId $SubscriptionId -KeyVaultName $secretsKeyVaultName`""
+# Creating a file to run to load environment variables
+$loadScriptDir = Join-Path $PSScriptRoot "..\..\..\..\.." -Resolve
```

Comment:
I chose the simplest method, which is knowing how deep this script is. If someone moves it to another folder, then this would likely break.